### PR TITLE
Biospecimen Dashboard: Replaced first name with preferred name in biospec survey reminders

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
Title^^
Note:
Cases where user has no preferred name than utilizes first name.

https://github.com/episphere/connectApp/issues/576

PoC:

Connect_ID: 2808118485
With no preferred name uses first name
Screenshot 2023-07-21 at 1 11 39 PM
Email:
Screenshot 2023-07-21 at 1 09 29 PM

Connect_ID: 2808118485
With preferred name
Screenshot 2023-07-21 at 1 20 19 PM
Email:
Screenshot 2023-07-21 at 1 18 42 PM

### CODE REVIEW

File: src/emailTemplates.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const baselineEmailTemplate = (data, isClinical) => {
     }
 
     return `
-        Dear ${data['399159511']},
+        Dear ${data['153211406'] !== '' ? data['153211406'] : data['399159511']},
```
Comment (we-ai) [inline src/emailTemplates.js:39]: How about `data['153211406'] || data['399159511']`?

**Review by we-ai — APPROVED** [review]

File: src/emailTemplates.js (hunk 1 above)
Comment (abhinavjonnada82) [reply src/emailTemplates.js:39]: this would execute `data['153211406']` if this is an empty string as well

Comment (JoeArmani) [reply src/emailTemplates.js:39]: I think prefName is null on signup when it isn't filled in by the participant, so either of the following would cover that case in addition to the case you've covered:

```javascript
(data['153211406'] !== '' && data['153211406'] !== null && data['153211406'] !== undefined) ? data['153211406'] : data['399159511']

//or the far more succinct (this covers null, undef, and empty string)
data['153211406'] ? data['153211406'] : data['399159511']
```

**Review by JoeArmani — CHANGES_REQUESTED** [review]
minor item (null case handling)

File: src/emailTemplates.js (hunk 1 above)
Comment (we-ai) [reply src/emailTemplates.js:39]: `A ? A : B` can be shortened to `A || B`.

Comment (abhinavjonnada82) [reply src/emailTemplates.js:39]: ill do ` data['153211406'] || data['399159511']`

Comment (abhinavjonnada82) [reply src/emailTemplates.js:39]: Good conversation guys!

**Review by JoeArmani — APPROVED** [review]
looks good

**Pull request merged**